projects/common/util/element.factory.ts

```diff
@@ -28,7 +27,7 @@ import { RadioGroupImagesElement } from '../models/compound-elements/radio-group
 import { DropListComponent } from '../element-components/compound-elements/drop-list.component';
 import { DropListElement } from '../models/compound-elements/drop-list';
 import { SliderElement } from '../models/slider-element';
-import {SliderComponent} from "../element-components/slider.component";
+import { SliderComponent } from '../element-components/slider.component';
```

Comment:
Nicht wirklich schlimm, aber sowas auch lieber in eigenem Commit. Zumindest erschließt sich mir der Zusammenhang mit "Add defaults for background, width and height" nicht. ;)